llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -247,8 +247,8 @@ def _validate_config(self) -> None:
                 self.attn_config['rope_imp']
                 == 'dail') and (self.attn_config['rope_dail_config']['type']
                                 not in ['original', 'xpos']):
-            raise NotImplementedError(
-                'If using dail implementation of rope, the type should be one of "original" or "xpos".'
+            raise ValueError(
+                'If using the dail implementation of rope, the type should be one of "original" or "xpos".'
```

Comment:
add using RoPE v ALiBi section in [tutorial](https://github.com/mosaicml/llm-foundry/blob/main/TUTORIAL.md#faqs)
Document the different configurations there.

Reply:
Added [here](https://github.com/ShashankMosaicML/llm-foundry/blob/rotary_hf_imp/TUTORIAL.md#what-kinds-of-positional-embeddings-does-llm-foundry-support)